docs/docs/build/custom-apis/custom-apis.md

```diff
@@ -18,9 +18,9 @@ type: api
 sql: SELECT abc FROM my_model
 ```
 
-### Using BigQuery or Snowflake as the OLAP Engine
+### Using Alternative OLAP Engines
 
-By default, SQL APIs execute queries against your default OLAP engine (typically DuckDB). However, you can specify a different OLAP engine using the `connector` parameter. This allows you to query data directly from BigQuery or Snowflake tables without ingesting them into Rill.
+By default, SQL APIs execute queries against your default OLAP engine (typically DuckDB). However, you can specify a different OLAP engine using the `connector` parameter. This allows you to query data directly from **BigQuery**, **Snowflake**, **MySQL**, **Postgres**, **Redshift**, or **Athena** without ingesting them into Rill.
```

Comment:
I'm not following in what sequence these are listed

<img width="799" height="516" alt="Image" src="https://github.com/user-attachments/assets/3101af95-2639-4741-8155-5f01e851fc48" />